sdk/cosmosdb/cosmos/samples-dev/ChangeFeedIterator/ChangeFeedFullFidelityEntireContainer.ts

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * @summary Demonstrates using a ChangeFeed for entire container with AllVersionsAndDeletes mode
+ */
+
+import * as dotenv from "dotenv";
+dotenv.config();
+
+import { finish, handleError, logSampleHeader, logStep } from "../Shared/handleError";
+import {
+  CosmosClient,
+  PartitionKeyDefinitionVersion,
+  Container,
+  StatusCodes,
+  ChangeFeedStartFrom,
+  ChangeFeedMode,
+  ChangeFeedIteratorOptions,
+} from "@azure/cosmos";
+
+const key = process.env.COSMOS_KEY || "<cosmos key>";
+const endpoint = process.env.COSMOS_ENDPOINT || "<cosmos endpoint>";
+const databaseId = process.env.COSMOS_DATABASE || "<cosmos database>";
+const containerId = process.env.COSMOS_CONTAINER || "<cosmos container>";
+
+logSampleHeader("Change Feed For Entire Container With AllVersionsAndDeletes Mode");
+
+async function ingestData(container: Container, initialize: number, end: number) {
```

Comment:
can we add these 2 methods(ingestData and insertAnsModifyData) in end?

Reply:
addressed